examples/quickstart.ipynb

```diff
@@ -0,0 +1,1146 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Started"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To install the most recent stable release for Modin run the following code on your command line:"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "pip install modin[all] "
```

Comment:
I do not see this line if I open this file via `view file` button. Is that expected?

Reply:
Seems like an [issue with Github rendering raw cells](https://github.com/microsoft/vscode-jupyter/issues/1466), switched to code cell instead.